addon/mixins/ember-hooks.js

```diff
@@ -1,3 +1,6 @@
+/* global clone */
+// TODO: figure out how to make clone not global
```

Comment:
Could we just use this?
```javascript
const clone = val => JSON.parse(JSON.stringify(val));
```

What is the benefit of pulling in the clone library?

Reply:
It handles circular references and promises, etc. But I agree it's kind of overkill here